From 8140d73d63383f22c9d88e5f3207b201cddebff2 Mon Sep 17 00:00:00 2001
From: Haiqing Bai <Haiqing.Bai@windriver.com>
Date: Wed, 6 Apr 2016 18:38:25 +0800
Subject: [PATCH] trousers: Conditional compile DES related code.

Added "OPENSSL_NO_DES" checking for DES related
code in trousers.

Upstream-Status: Pending

Signed-off-by: Haiqing Bai <Haiqing.Bai@windriver.com>
---
 src/trspi/crypto/openssl/symmetric.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/trspi/crypto/openssl/symmetric.c b/src/trspi/crypto/openssl/symmetric.c
index f5c3836..8ba183d 100644
--- a/src/trspi/crypto/openssl/symmetric.c
+++ b/src/trspi/crypto/openssl/symmetric.c
@@ -162,6 +162,7 @@ get_openssl_cipher(UINT16 alg, UINT16 mode)
 					break;
 			}
 			break;
+#ifndef OPENSSL_NO_DES
 		case TSS_ALG_DES:
 		case TCPA_ALG_DES:
 			switch (mode) {
@@ -202,6 +203,7 @@ get_openssl_cipher(UINT16 alg, UINT16 mode)
 					break;
 			}
 			break;
+#endif /* OPENSSL_NO_DES */
 		case TPM_ALG_AES192:
 		case TSS_ALG_AES192:
 			switch (mode) {
-- 
1.9.1

